### PR TITLE
console: update select font for firefox (close #3955)

### LINKED ISSUE #3955: Console dropdowns are serif in Firefox
1.1.0-beta2, the dropdowns look like this in FF:

Screen Shot 2020-02-25 at 5 38 00 PM
Screen Shot 2020-02-25 at 5 37 51 PM

Would be nice if they were sans-serif like in Chrome. Much easier to read that way.

### PR DESCRIPTION
**Pull request by kimberrypi**
### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [ ] Server
- [x] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

### Related Issues
closes #3955

### Solution and Design
added `font-family: sans-serif;` to bootstrap overrides and react table overrides

### Steps to test and verify
*use firefox to check
1. `http://localhost:3000/data/schema/public`, click the dropdown
2. select a table and click the dropdown below filter
3. click the dropdown in the pagination

all should have sans-serif font

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [x] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [x] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [x] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [x] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - console: remove ONLY as default for ALTER TABLE in column alter operations (close #5512) #5706
 - console: add option to flag an insertion as a migration from `Data` section (close #1766) (#4933)
 - console: add `<3 hasura` section to view updates and notifications from Hasura (#5070)
+- console: fix select font family in firefox (#3955)
```
Comment (beerose) [inline CHANGELOG.md:66]:
```suggestion
- console: fix select font family in firefox (close #3955) (#5923)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - console: remove ONLY as default for ALTER TABLE in column alter operations (close #5512) #5706
```
Comment (beerose) [inline CHANGELOG.md:63]:
```suggestion
- console: remove ONLY as default for ALTER TABLE in column alter operations (close #5512) (#5706)
```

**Review by beerose — APPROVED** [review]
Thanks for your PR! 😊

Approving console changes

**Review by tirumaraiselvan — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -62,9 +62,10 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 **NOTE:** If you have event triggers with names greater than 42 chars, then you should update their names to avoid running into Postgres identifier limit bug (#5786)
 - console: allow user to cascade Postgres dependencies when dropping Postgres objects (close #5109) (#5248)
 - console: mark inconsistent remote schemas in the UI (close #5093) (#5181)
-- console: remove ONLY as default for ALTER TABLE in column alter operations (close #5512) #5706
+- console: remove ONLY as default for ALTER TABLE in column alter operations (close #5512) (#5706)
 - console: add option to flag an insertion as a migration from `Data` section (close #1766) (#4933)
 - console: add `<3 hasura` section to view updates and notifications from Hasura (#5070)
+- console: fix select font family in firefox (close #3955) (#5923)
```
Comment (beerose) [inline CHANGELOG.md:68]:
```suggestion
```

**Review by beerose — CHANGES_REQUESTED** [review]
Could you actually change the font to `'Gudea'`? We're using this font in the application, and it was the default in browsers other than Firefox before.

**Comment by kimberrypi** [comment]
> Could you actually change the font to `'Gudea'`? We're using this font in the application, and it was the default in browsers other than Firefox before.

I updated it to be 'Gudea' :)

**Comment by mehtavishwa30** [comment]
Hello @kimberrypi!

Thanks so much for your contribution to Hasura OSS during the Hacktoberfest 2020! We'd love to send you some Hasura swag as a token of appreciation and you will be getting an email with the swag form shortly. ✨

**Comment by beerose** [comment]
PR was merged in https://github.com/hasura/graphql-engine/commit/0c1133b3933e6e9362231869551235fd73403a69

**Pull request closed without merging**